my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+export const CustomInput = props => {
+  const { label, input, meta } = props;
+  return (
+    <Fragment>
+      <label> {props.label} </label>
+      <input {...props.input} type={props.type} />
+      {meta.error && meta.touched && !meta.active && <div> {props.label + meta.error}</div>}
```

Comment:
Remove the `&& !meta.active`. That way, you show the error when the user is typing and he doesn't forget what the error was. Keep in mind some error messages are lengthier than others. For example, if the password has many specific characters requirement, you want the user to see the error while he's fixing it.